package/python-kiwi-spec-template

```diff
@@ -308,6 +308,24 @@ This package contains the basic PXE directory structure which is
 needed to serve kiwi built images via PXE.
 %endif
 
+%package -n kiwi-dracut-iso
```

Comment:
Please rename this to `dracut-kiwi-live`, as that follows the conventions of dracut module packaging in Fedora, openSUSE, and Mageia.

Reply:
ok thanks for the hint, will do